src/Metrics.tcl

```diff
@@ -254,5 +254,245 @@ proc report_design_area_metrics {args} {
   utl::metric_float "design__instance__utilization__stdcell" $stdcell_util
 }
 
+proc report_puts { out } {
+  upvar 1 when when
+  upvar 1 filename filename
+  set fileId [open $filename a]
+  puts $fileId $out
+  close $fileId
+}
+
+define_cmd_args "report_metrics" {[-stage][-when][-include_erc][-include_clock_skew][-metrics_report_dir dir]}
+proc report_metrics { stage when {include_erc true} {include_clock_skew true} {metrics_report_dir "."}} {
```

Comment:
this should be using: https://github.com/The-OpenROAD-Project/OpenROAD/blob/42ba30d730851a73bd2996c5673a05fd79520f87/docs/contrib/TclFormat.md?plain=1#L17